owasp-dependency-check-suppressions.xml

```diff
@@ -391,6 +391,16 @@
     <packageUrl regex="true">^pkg:maven/org.apache.logging.log4j/log4j-core@2.17.1$</packageUrl>
     <cve>CVE-2022-33915</cve>
   </suppress>
+  <suppress>
+       <!--
+         - TODO: The lastest version of ambari-metrics-common is 2.7.0.0.0, released in July 2018.
```

Comment:
whats the todo here? As in, whats the future work?

Reply:
I had copied it over as I believe the intent was to upgrade ambari-metrics-common to a later when and if it became available. I've removed the TODO now